consensus/polybft/bridge_manager.go

```diff
@@ -318,27 +304,27 @@ func (b *bridgeManager) Close() {
 	b.exitEventRelayer.Close()
 }
 
-// initStateSyncManager initializes state sync manager
-// if bridge is not enabled, then a dummy state sync manager will be used
-func (b *bridgeManager) initStateSyncManager(
+// initBridgeEventManager initializes bridge event manager
+// if bridge is not enabled, then a dummy bridge event manager will be used
+func (b *bridgeManager) initBridgeEventManager(
 	bridgeBackend BridgeBackend,
 	runtimeConfig *runtimeConfig,
 	logger hclog.Logger) error {
-	stateSyncManager := newStateSyncManager(
+	bridgeEventManager := newBridgeEventManager(
 		logger.Named("state-sync-manager"),
 		runtimeConfig.State,
-		&stateSyncConfig{
+		&bridgeEventManagerConfig{
 			key:               runtimeConfig.Key,
-			dataDir:           runtimeConfig.DataDir,
 			topic:             runtimeConfig.bridgeTopic,
-			maxCommitmentSize: maxCommitmentSize,
+			maxNumberOfEvents: maxNumberOfEvents,
 		},
 		bridgeBackend,
+		1,
```

Comment:
Do we need to have a separate `bridgeEventManager` for each chain? At least add a comment to remove this 1 at some point later.